$NetBSD: patch-CMakeLists.txt,v 1.1 2011/02/17 09:20:17 markd Exp $

ADM_BSD_FAMILY is a superset of APPLE and still not Linux

--- CMakeLists.txt.orig	2010-10-23 16:12:07.000000000 +0000
+++ CMakeLists.txt
@@ -58,7 +58,7 @@ IF (ADM_CPU_ALTIVEC)
 	SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${ADM_ALTIVEC_FLAGS}")
 ENDIF (ADM_CPU_ALTIVEC)
 
-IF (UNIX AND NOT APPLE)
+IF (UNIX AND NOT ADM_BSD_FAMILY)
 	# jog shuttle is only available on Linux due to its interface
 	SET(USE_JOG 1)
 ENDIF (UNIX AND NOT APPLE)
